test/Twig/Tests/Extension/FilesizeTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2015 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+require_once __DIR__.'/../../../../lib/Twig/Extensions/Extension/Filesize.php';
```

Comment:
Some other tests cannot rely on autoloading because they call functions rather than using the class